app/assets/v2/js/grants/create-collection-modal.js

```diff
@@ -48,15 +45,12 @@ Vue.component('create-collection-modal', {
     },
     createCollection: async function() {
       const csrfmiddlewaretoken = document.querySelector('[name=csrfmiddlewaretoken]').value;
-      const checkedOut = CartData.loadCheckedOut();
-      const cart = (checkedOut.length > 0 ? checkedOut : CartData.loadCart());
-      const grantIds = cart.map(grant => grant.grant_id);
```

Comment:
`grantIds` was an array here but `this.grantIds` is a string, have you tested the collection creation flow?

Reply:
oops! nice catch @gdixon 

thanks